Data/ByteString/Builder/Internal.hs

```diff
@@ -1052,27 +1063,35 @@ safeStrategy firstSize bufSize =
     nextBuffer Nothing             = newBuffer $ sanitize firstSize
     nextBuffer (Just (_, minSize)) = newBuffer minSize
 
+-- | Execute a 'Builder' and return the generated chunks as a 'L.LazyByteString'.
+-- The work is performed lazy, i.e., only when a chunk of the 'L.LazyByteString'
+-- is forced.
+{-# NOINLINE toLazyByteString #-} -- ensure code is shared
+toLazyByteString :: Builder -> L.LazyByteString
+toLazyByteString = toLazyByteStringWith
+    (safeStrategy L.smallChunkSize L.defaultChunkSize) L.Empty
+
 -- | /Heavy inlining./ Execute a 'Builder' with custom execution parameters.
 --
 -- This function is inlined despite its heavy code-size to allow fusing with
 -- the allocation strategy. For example, the default 'Builder' execution
--- function 'Data.ByteString.Builder.toLazyByteString' is defined as follows.
+-- function 'Data.ByteString.Builder.Internal.toLazyByteString' is defined as follows.
```

Comment:
Have you checked how this link renders in `Data.ByteString.Builder.Extra`? It would be nice to continue to link to the more canonical home `Data.ByteString.Builder` from that module if it's easily possible to do so.

Reply:
It seems to reference where it was imported from

```html
<p>This function is inlined despite its heavy code-size to allow fusing with
 the allocation strategy. For example, the default <code><a href="Data-ByteString-Builder.html#t:Builder" title="Data.ByteString.Builder">Builder</a></code> execution
 function <code><a href="Data-ByteString-Builder.html#v:toLazyByteString" title="Data.ByteString.Builder">toLazyByteString</a></code> is defined as follows.</p>
```
[Data-ByteString-Builder-Extra.zip](https://github.com/user-attachments/files/15918477/Data-ByteString-Builder-Extra.zip)